sdk/tests/deqp/framework/delibs/debase/deMath.js

```diff
@@ -48,6 +48,24 @@ deMath.deInBounds32 = function(a, mn, mx) {
 deMath.deFloatFrac = function(a) { return a - Math.floor(a); };
 
 /**
+ * Global variable used in function deMath.toFloat32()
+ */
+deMath.FLOAT32ARRAY1 = new Float32Array(1);
+
+/**
+ * Transform a 64-bit float number into a 32-bit float number.
+ * Native dEQP uses 32-bit numbers, so sometimes 64-bit floating numbers in JS should be transformed into 32-bit ones to ensure the correctness of the result.
+ * @param {number} a
+ * @return {number}
+ */
+deMath.toFloat32 = function(a) {
+    deMath.FLOAT32ARRAY1[0] = a;
+    return deMath.FLOAT32ARRAY1[0];
+};
```

Comment:
Transform a 64-bit float number into a 32-bit float number.
Native dEQP uses 32-bit numbers, so sometimes 64-bit floating numbers in JS should be transformed into 32-bit ones to ensure the correctness of the result.